star-ui/src/components/CameraFeed.tsx

```
@@ -0,0 +1,64 @@
+                alignItems: 'center', justifyContent: 'center',
+                position: 'relative',
+                background: 'rgba(255, 255, 255, 0.02)',
+            }}>
+                <div style={{ textAlign: 'center', color: COLORS.textMuted }}>
+                    <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        width="48" height="48" viewBox="0 0 24 24"
+                        fill="none" stroke="currentColor" strokeWidth="1.5"
+                        strokeLinecap="round" strokeLinejoin="round"
+                        style={{ margin: '0 auto 12px auto', opacity: 0.5 }}
+                    >
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)